### PR TITLE
Version 2.0.0-alpha.2

### PR DESCRIPTION
**Pull request by windymilla**
Add changelog with brief description of alpha.1 features, and changes in alpha.2

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# Changelog
+
+
+## Version 2.0.0-alpha.2
+
+- Font family and size selection in Preferences (Edit menu)
```
Comment (srjfoo) [inline CHANGELOG.md:6]: It's not in the `Edit` menu for Macs. It's in the Application menu. In this case, `Python`, hopefully if we can package it properly, under the `Guiguts` menu once we get a formal release.

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+- Fixup Footnotes (Tools menu) - begun but not complete
+- Rewrap (Tools menu) - using "greedy" algorithm
+- Unmatched Markup features  (Tools menu)
+- Convert Fractions (Tools menu)
+- Normalize Characters (Tools menu)
+- Add/Remove Page Marker Flags (File-->Project menu) for transfer between
+  editors
+- Compose Sequences (Tools menu, and via Cmd+I/Ctrl+I shortcut) - only
+  accented characters and Unicode ordinals. List of sequences (Help Menu)
+- Message log (View menu)
```
Comment (srjfoo) [inline CHANGELOG.md:53]: Do you want to mention the `-d` option, or do you figure that's not something this group will be interested in? Are they still bringing it up from the command line?

**Review by srjfoo — APPROVED** [review]
My comments are just questions. Feel free to ignore if they're not relevant here.

File: CHANGELOG.md (hunk 2 above)
Comment (windymilla) [reply CHANGELOG.md:53]: I don't think it's really needed, but I will mention it along with the -r option

File: CHANGELOG.md (hunk 1 above)
Comment (windymilla) [reply CHANGELOG.md:6]: I'll edit it to mention the Python menu on Macs.

**Pull request merged**